server/routes/filmsRoutes.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Film = mongoose.model('films');
+const User = mongoose.model('users');
+
+module.exports = (app) => {
+  app.get(`/api/films/:pageId/`, async (req, res) => {
+    const { pageId } = req.params
+    i = 5
+    y = i * (pageId - 1)
```

Comment:
It is unclear to me what these variables look like? Could you give them a name that matches their intent?

to me it looks like `i` is `perPage` and `y` is `pageOffset`